File: app/src/main/java/com/isolity/toastalarm/model/TimeOfDay.java
package com.isolity.toastalarm.model;

/**
 * Created by shohei52a on 2017/04/22.
 */

public class TimeOfDay {
    private int hour;
    private int minute;

    public TimeOfDay(int h, int m) {
        hour = h;
        minute = m;
    }

    public String toString() {
        return   String.valueOf(hour) + ':' + String.valueOf(minute);
    }
}

File: app/src/main/java/com/isolity/toastalarm/model/TimeOfDay.kt
package com.isolity.toastalarm.model

/**
 * Created by shoarai on 2017/04/22.
 */

class TimeOfDay( val hour: Int, val minute: Int) {

    override fun toString(): String {
        return String.format("%02d", hour) + ":" + String.format("%02d", minute)
    }
}
